apps/api/src/queries/buildViewModelForResource/viewModels/term.view-model.ts

```diff
@@ -21,13 +23,33 @@ import { Maybe } from '../../../lib/types/maybe';
 import { NotFound } from '../../../lib/types/not-found';
 import { clonePlainObjectWithOverrides } from '../../../lib/utilities/clonePlainObjectWithOverrides';
 import cloneToPlainObject from '../../../lib/utilities/cloneToPlainObject';
+import { CoscradDataExample } from '../../../test-data/utilities';
 import { DeepPartial } from '../../../types/DeepPartial';
 import { DTO } from '../../../types/DTO';
 
 /**
  * This is the first view model leveraging a new approach that involves denormalized,
  * event-sourced, materialized views.
  */
+@CoscradDataExample<TermViewModel>({
```

Comment:
Woohoo! I love this.

Reply:
BTW You can register multiple examples by calling the decorator multiple times. Not sure why I did it that way instead of just having `{examples}` instead of `{example}`.